tests/common/gcp_api/appengine_test.py

```diff
@@ -38,7 +38,7 @@ class AppEngineTest(unittest_utils.ForsetiTestCase):
     def setUpClass(cls, mock_google_credential):
         """Set up."""
         fake_global_configs = {
-            'max_appengine_api_calls_per_second': 20}
+            'appengine': {'max_calls': 18, 'period': 1}}
```

Comment:
You could, here and below, replace the specific key with the global API_NAME as mentioned in the earlier comment.

`ae.AppEngineClient.API_NAME: '{...}'`